bstrp_functions.sh

```diff
@@ -279,7 +280,7 @@ show_supported_features() {
 
 read_feature_options(){
   local choice
-  read -p "Enter choice [ A - T or 1-3 ] " choice
+  read -p "Enter choice [ A - T or 1-4 ] " choice
```

Comment:
Should be A - V :)

Reply:
Easy change in https://issues.apache.org/jira/browse/MINIFICPP-880.